docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,18 +4,24 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
-AddressBook *-right-> "1" UniqueTagList
-UniqueTagList -[hidden]down- UniquePersonList
+StaffBook *-right-> "1" UniquePersonList
+StaffBook *-right-> "1" UniqueTagList
+StaffBook *-right-> "1" UniqueMeetingList
+UniqueMeetingList -[hidden]down- UniqueTagList
 UniqueTagList -[hidden]down- UniquePersonList
 
 UniqueTagList -right-> "*" Tag
+UniqueMeetingList -right-> "*" Meeting
 UniquePersonList -right-> Person
 
+Person --> "*" Availability
```

Comment:
should this maybe be moved below as it is an attribute of `Person`?
should the arrow also be `*-->` instead? (to demonstrate "composition")

Reply:
Fixed

Reply:
I think the .png has not been updated

Reply:
my bad, updated the png for BetterModelClassDiagram and CommityActivityDiagram